### ISSUE TITLE
[a11y]: ai-chat should set meaningful focus after feedback is submitted

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon-labs#867]
### Package version

0.18.0

### React version

18.3.1

### Automated testing tool and ruleset

n/a

### Assistive technology

JAWS

### Description

After filling out the feedback form and submitting the feedback, the keyboard focus is getting lost.

Recommended fix: Ensure after user clicks on submit button, the keyboard focus should moved to a meaningful element.

User impact: Keyboard and screen reader users will affects with the further interaction.

### WCAG 2.1 Violation

WCAG Mapping: 2.4.3 Focus Order

### Reproduction/example

n/a

### Steps to reproduce

1. for a response message, select thumbs up or down
2. fill out the feedback form
3. select submit
4. see that the focus is lost

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
